MAINTAINERS.md

```diff
@@ -1,21 +1,25 @@
 # Maintainers
 
-MAINTAINERS:
+Code for this project is reviewed by any other member of the team before it is merged into the project. Team members include:
 
-- lucasmbrown-usds
-- aprilypchen
+## Project Leadership
 
-approvers:
+* Lucas Brown
 
-- TBA
+## Engineering
```

Comment:
Let's include folks' github usernames, so we match those to the correct person on the Maintainers list.

Reply:
I based this off of https://github.com/usds/justice40-tool/blob/main/MAINTAINERS.md format, which I think does a nice job of keeping thing simple, while linking to the code owners file.